tests/run-make/linker-warning/rmake.rs

```diff
@@ -61,7 +61,7 @@ fn main() {
         diff()
             .expected_file("short-error.txt")
             .actual_text("(linker error)", out.stderr())
-            .normalize(r#"/rustc[^/_-]*/"#, "/rustc/")
+            .normalize(r#"^/rustc[^/_-]*/"#, "/rustc/")
```

Comment:
For me locally, the test as a whole passes just fine if I remove this normalization step altogether but I guess there are some environments where there's a `rustcXXX` (tempdir?) in the (fake?) rootdir?

Reply:
Right, that was what this was supposed to normalize. I was thinking about the `/rustc/<commit>/` for source files, but those wouldn't have matched anyway.

Reply:
Okay, so I've now completely remove that normalization step because `short-error.txt` doesn't contain any occurrence of `/rustc/` so I'm led to believe that it never fired anyway (except on my machine) since the subsequent normalizations don't depend the ↦`/rustc/` normalization I think (so the lack of `/rustc/` should really mean "never fired").

Does that make sense to you or am I misunderstanding something? In which environments or circumstances would it be possible for rustc to pass a `/rustc/<commit>/` path to the linker (and even if it that the test would fail I believe)?

Reply:
I think the `.normalize(r#""[^"]*\/symbols.o""#, "\"/symbols.o\"").normalize(r#""[^"]*\/raw-dylibs""#, "\"/raw-dylibs\"")` below made the tempdir normalization obsolete.